tikv-client/src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -193,6 +196,42 @@ public String readUTF() {
     }
   }
 
+  /**
+   * peek the first encoded value and return its length
+   *
+   * @return first encoded value
+   */
+  public int cutOne() {
+    if (available() < 1) {
+      throw new CodecException("invalid encoded key");
+    }
+    int flag = readByte();
+    int l;
+    switch (flag) {
+      case NULL_FLAG:
+      case INT_FLAG | UINT_FLAG | FLOATING_FLAG | DURATION_FLAG:
+        l = 8;
+        break;
+      case BYTES_FLAG:
+        l = readByte();
+        break;
+        //      case COMPACT_BYTES_FLAG:
```

Comment:
COMPACT_BYTES_FLAG and DECIMAL_FLAG are not annotated.
Check if we can do that to